CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Within V2
 
+## 2.0.0-RC.1
+ - Flash Messages are not translated in the Controllers, not in views anymore. If you have custom Flash Messages, translate them in your Controller instead of the view.
```

Comment:
Flash messages **are** translated in the controllers.  😄

Reply:
oops :D

Reply:
fixed